test/protocol/DisputeResolverHandlerTest.js

```diff
@@ -2454,6 +2454,27 @@ describe("DisputeResolverHandler", function () {
           );
       });
 
+      it("Should not emit 'DisputeResolverUpdateApplied' event if caller doesn't especify any field", async function () {
+        disputeResolver.operator = other1.address;
+        await accountHandler.connect(admin).updateDisputeResolver(disputeResolver);
+
+        await expect(accountHandler.connect(other1).optInToDisputeResolverUpdate(disputeResolver.id, [])).to.not.emit(
+          accountHandler,
+          "DisputeResolverUpdateApplied"
+        );
+      });
+
+      it("Should not emit 'DisputeResolverUpdateApplied'event if there is no pending update for especified field", async function () {
```

Comment:
```suggestion
      it("Should not emit 'DisputeResolverUpdateApplied' event if there is no pending update for specified field", async function () {
```

Reply:
fixed

Reply:
There is still a space missing between `'` and `event`.

Since you need to push another change anyway (to fix failing test), can you please also change this as well.